### ISSUE TITLE
CF Login fails: Port 443 Connection Refused

### ISSUE THREAD

**Issue by jallwrig** [cloudfoundry-incubator/cfdev#11]
```
cf login -a https://api.v3.pcfdev.io --skip-ssl-validation
API endpoint: https://api.v3.pcfdev.io
FAILED
Error performing request: Get https://api.v3.pcfdev.io/v2/info: dial tcp 10.144.0.34:443: getsockopt: connection refused
```

**Comment by ekcasey** [comment]
Hi @jallwrig , you and a handful of other users have discovered issue in the alpha release of CF Dev. CF Dev attempts to forward `10.144.0.34:80` and `10.144.0.34:443` on the guest to `10.144.0.34:80` and `10.144.0.34:443` on the host. However, `vpnkit` does not have permissions to bind those privileged ports. The reason we didn't catch this issue sooner is that machines with Docker for Mac installed have modified configurations which makes this work.

If you don't mind installing Docker for Mac, you can do that to get the current alpha working. In the near term future we will be releasing a version of CF Dev that does not attempt to bind privileged ports. In the long run we may opt to take  steps similar to Docker for Mac which will allow us to bind `10.144.0.34:80` and `10.144.0.34:443`.

**Comment by ssisil** [comment]
Fixed in release [v0.0.2](https://github.com/cloudfoundry-incubator/cfdev/releases/tag/v0.0.2)
